### PR TITLE
Adding views,tests and template for Board list view

### PR DESCRIPTION
**Pull request by SourabhTestpress**
Created views,tests and template for board list view. Wrote 2 test cases, to verify working.

**Stats**

    Number of Test Cases - 2
    Number of SQL Queries - 1

**Guidelines**

PEP8(Python)/Testpress JS compatible Yes
Is each logical block separated with a line break? Yes
Have you self reviewed this PR in context to the previous PR feedbacks? Yes

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+  <body>
+    {% block body %}
+    {% endblock %}
+
+
+     <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script
+              src="https://code.jquery.com/jquery-3.5.1.min.js"
```
Comment (waranlogesh) [inline templates/base.html:23]: use proper formatting

File: templates/boards/board_list.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	</table>
+</div>
+
+{% endblock %}
+
+
+{% block scripts %}
+<script type="text/javascript">
+
+$(document).ready(function($) {
```
Comment (waranlogesh) [inline templates/boards/board_list.html:53]: Use href directly. No need for javascript for this stuff.

File: boards/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# imports from this folder
+from ..views import board_list
+
+
+class HomepageTests(TestCase):
+    """It tests list of Boards"""
+
+    def test_home_view_status_code(self):
+        url = reverse("home")
+        response = self.client.get(url)
```
[the remark this answers is not in the document]
Comment (waranlogesh) [reply boards/tests/test_views.py:14]: self.assertEquals(expected_value, obtained_value) so this should be self.assertEquals(200, response.status_code)

**Review by waranlogesh — CHANGES_REQUESTED** [review]
Add unit test to check the template used in the view.
Add test case for empty view.
Add test to check if 10 boards are present whether its displaying all 10.
